gangplank/ocp/ocp.go

```diff
@@ -56,6 +63,125 @@ func ocpBuildClient() error {
 		log.Infof("Using %s as working directory.", cDir)
 		cosaSrvDir = cDir
 	}
+	return nil
+}
+
+// getPushTagless returns the registry, and push path
+// i.e registry.svc:5000/image/bar:tag returns "registry.svc:5000" and "image/bar"
+func getPushTagless(s string) (string, string) {
+	pushParts := strings.Split(s, "/")
+	// split the name and tag
+	splitLast := strings.Split(pushParts[len(pushParts)-1], ":")
+	// strip off the tag, and replace it with just the name
+	pushPathParts := strings.Split(strings.TrimRight(s, strings.Join(splitLast, ":"))+splitLast[0], "/")
+	return pushParts[0], strings.Join(pushPathParts[1:], "/")
+}
+
+// uploadCustomBuildContainer implements the custom build strategy optional step to report the results
+// to the registry as an OCI image.
+func uploadCustomBuildContainer(ctx ClusterContext, apiBuild *buildapiv1.Build, buildID string) error {
+	if apiBuild == nil || apiBuild.Spec.Strategy.CustomStrategy == nil || apiBuild.Spec.Output.To == nil {
+		log.Debug("Build is not running as a custom build strategy or output name is not defined")
+		return nil
+	}
+
+	baseEnv := append(os.Environ(),
+		"DISABLE_TLS_VERIFICATION=1",
```

Comment:
We only need this when running in an environment with custom CA certs though right?  IOW for the OpenShift-Prow-CI use case it would seem to be good to keep this on.

Maybe we could somehow derive this automatically from the config git repo, or some other global Gangplank configuration?

Reply:
Indeed, that's an excellent idea. I've pushed with this being a jobspec variable:
```
job:
    disable_tls_verification: true
```